### PR TITLE
NextJS 14 `next export` has been removed

### LINKED ISSUE #580: Next 14 Support
Next 14 has changed how to export static site:

https://nextjs.org/docs/pages/building-your-application/deploying/static-exports#version-history

### PR DESCRIPTION
**Pull request by shivam-sharma7**
## Describe your changes
check-https://nextjs.org/docs/pages/building-your-application/deploying/static-exports#version-history

Fix #580

### CODE REVIEW

**Comment by melloware** [comment]
I think we need to update the docs too: https://docs.quarkiverse.io/quarkus-quinoa/dev/web-frameworks.html#nextjs-config

and test it. Someone else on another forum I am on was saying Next14 doesn't generate an `index.html` like the old export did which Quinoa needs?

**Comment by shivam-sharma7** [comment]
the next export command is deprecated in Next.js 14. Starting with this version, we should use the "output": "export" option in your `next.config.js` file to generate static exports.

Here is an example of my merged PR- https://github.com/tailwarden/komiser/pull/1217/files

**Comment by melloware** [comment]
Yes so we need to update the docs for Next.JS or check for the `output: export` and throw an error if not found.

**Review by melloware — CHANGES_REQUESTED** [review]
See comments

**Comment by shivam-sharma7** [comment]
@melloware We don't need to check I think because that is totally depend on user deployment.So we need to update the docs.

**Comment by melloware** [comment]
When I tried this before Quinoa does NOT work unless its in this mode so we have to check and make it required for Next.   You can't use Next with Quinoa without the static export mode.

File: docs/modules/ROOT/pages/web-frameworks.adoc [hunk 1]
```diff
@@ -206,6 +206,13 @@ This will configure the build to export the static files:
   }
 ----
 
+And in new version of next.js you need to add the following property in `next.config.js` file:
```
Comment (melloware) [inline docs/modules/ROOT/pages/web-frameworks.adoc:209]:
```suggestion
In Next.js 14+ you need to add the following property in `next.config.js` file:
```

**Review by melloware — CHANGES_REQUESTED** [review]

**Pull request merged**